examples/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/qcbm/qcbm_job.py

```diff
@@ -31,10 +30,16 @@ def main():
     print(f"Using device {device}")
 
     print("Starting circuit training...")
-    params = train_circuit(device, hyperparams, data)
+    params, cost_tracker = train_circuit(device, hyperparams, data)
```

Comment:
Rather than returning the cost tracker from `train_circuit`, we can have two separate cost trackers, one started in main to save the summary to the job result, and a second one confined to `train_circuit` for the purpose of logging.

Reply:
okay, updated